app/components/search.tsx

```diff
@@ -106,6 +108,17 @@ export default function Search({
                 }}
               />
             ))}
+          <a
+            href={`https://stampy.ai/wiki/Special:FormStart?form=Q&page_name=${searchInput}`}
+            target='_blank' rel="noreferrer"
```

Comment:
could you please check `npm run lint` and/or enable eslint and prettier in your editor? JSX props should be in double quotes in our code style settings, I believe